.github/workflows/Intel.yml

```diff
@@ -38,52 +38,28 @@ jobs:
         sudo apt-get install intel-oneapi-dev-utilities intel-oneapi-mpi-devel intel-oneapi-openmp intel-oneapi-compiler-fortran intel-oneapi-compiler-dpcpp-cpp-and-cpp-classic
         echo "source /opt/intel/oneapi/setvars.sh" >> ~/.bash_profile
 
-    - name: cache-bacio
-      id: cache-bacio
+    - name: cache-w3nco
+      id: cache-w3nco
```

Comment:
We don't want to use w3nco, everything from w3nco is also included in w3emc.

Reply:
Back to dependency on w3emc and bacio in lieu of w3nco?